pkg/server/v3/server_test.go

```diff
@@ -154,7 +155,8 @@ var (
 	endpoint        = resource.MakeEndpoint(clusterName, 8080)
 	cluster         = resource.MakeCluster(resource.Ads, clusterName)
 	route           = resource.MakeRoute(routeName, clusterName)
-	listener        = resource.MakeHTTPListener(resource.Ads, listenerName, 80, routeName)
+	scopedRoute     = resource.MakeScopedRoute(scopedRouteName, routeName, []string{"127.0.0.1"})
+	listener        = resource.MakeScopedRouteHTTPListener(resource.Ads, listenerName, 80, scopedRouteName)
```

Comment:
Do we no longer test a normal `HttpRouteListener`?

Reply:
I'll update the integration test again to make sure we test both, but yes you are correct I removed it thinking it didn't matter :)